src/routes/appRouter.ts

```diff
@@ -0,0 +1,17 @@
+import express from 'express';
+import * as appController from '../controller/appController';
+
+const appRouter = express.Router();
+
+appRouter.get('/getBoard/:id', appController.getBoardController);
+appRouter.get('/getNonMemberUsers/:id', appController.getNonMemberUsersController);
+
+appRouter.post('/createBoard/:id', appController.board);
+appRouter.post('/createCard/:id', appController.createCardController);
+appRouter.post('/createList/:id', appController.createListController);
+appRouter.post('/addMembersInBoard/:id', appController.addMembersinBoardController);
+
+appRouter.put('/updateListLevel/:id', appController.updateListLevelController);
+appRouter.put('/updateCardOwner/:id', appController.updateCardOwnerController);
```

Comment:
For REST APIs go through this doc:

- Use hyphenated-lower-case, now, and plural
- Use verb only if your expose API is executable functions

Example: In the case of `updateCardOwner` simply `cards` is enough

- https://www.coding-guidelines.lftechnology.com/docs/rest-api/naming-convention